driver/ppm_fillers.c

```diff
@@ -47,6 +47,18 @@ or GPL2.txt for full copies of the license.
 #include <linux/bpf.h>
 #endif
 
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 6, 0)
+struct compat_timespec {
+	int32_t tv_sec;
+	int32_t tv_nsec;
+};
+
+struct timeval {
+	int32_t tv_sec;
+	int32_t tv_usec;
+};
+#endif
```

Comment:
I don't quite understand what's going on here. Is `compat_timespec` actually used anywhere? And is this `timeval` struct just kicking the can down the road by re-introducing a 32-bit time structure?

Reply:
They're both parts of the userspace API